client/modules/components/src/Card.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
```

Comment:
What is a "card" here?

Reply:
"Card" is a common name for a UI panel of some kind. They're very generic.

https://open-ui.org/components/card.research/